src/rez/package_order.py

```diff
@@ -125,12 +253,19 @@ def to_pod(self):
 
            type: sorted
            descending: true
+           packages: ["foo"]
         """
-        return {"descending": self.descending}
+        return {
+            "descending": self.descending,
+            "packages": self.packages,
+        }
 
     @classmethod
     def from_pod(cls, data):
-        return cls(descending=data["descending"])
+        return cls(
+            descending=data["descending"],
```

Comment:
```suggestion
            data["descending"],
```

Reply:
This is fine, but I do think that it is a bit easier to understand with the kwarg, as it originally in there.

Reply:
The thing is that the first argument the class takes is a positional argument. passing a keyword argument instead could lead someone to think the arguments can be freely reordered. Passing a keyword argument for a positional argument is a bad practice.

Reply:
What makes it positional? It has a default value (e.g. `SortedPackageOrder()` works fine). 

I've heard that it's bad practice to provide positional arguments to what are kwargs, but not the other way around, especially if it provides better readability.  If it were actually positional, then maybe I'd agree, but even then I think it's clearer as a consumer of the API about what is actually going on.

Obviously this is an academic thing since the key of the data is a strong hint, but it is certainly more readable to have it explicit about which data is slotted into which key.  Additionally, the data could be reordered freely and it would work fine.

Since the original code had those as kwargs, I'd lean towards leaving it as they were, since we lose nothing and gain a small amount of legibility and reliability for fewer changes.

I won't die on this hill though, so if you really want it, I won't push back on it any more.

Reply:
We are in the `SortedOrder` class, which has `def __init__(self, descending, packages=None):`. `descending` is positional.

Reply:
Oh, I see.  Quite right.